CMakeLists.txt

```diff
@@ -317,17 +308,19 @@ if (${use_prov_client_core})
         add_subdirectory(provisioning_service_client)
     endif()
 
+    message("Main CMakeLists: Line 311: adding provisioning_client")
     add_subdirectory(provisioning_client)
 endif()
 
 if (${build_service_client})
+    message("Main CMakeLists: Line 316: adding iothub_service_client")
     add_subdirectory(iothub_service_client)
 endif()
 
 if (${run_e2e_tests} OR ${run_longhaul_tests} OR ${nuget_e2e_tests} OR ${run_sfc_tests})
     add_subdirectory(testtools)
 endif()
-
+message("Main CMakeLists: Line 322: add_subdirectory iothub_client")
```

Comment:
You might want to do a cleanup pass on these debug messages

Reply:
yes this pr is just so I can look at the differences... and it's between your fork and my vcpkg-patch branch